src/Repository/Entity/Repository.php

```diff
@@ -0,0 +1,251 @@
+<?php
+
+namespace OAT\DependencyResolver\Repository\Entity;
+
+class Repository implements \JsonSerializable
+{
+    public const CSV_TITLES = [
```

Comment:
CSV related stuff should not be here on the entity level...

Reply:
Moved to RepositoryMapConverter but it makes the fields of Repository and the titles not sit in the same place... well, no big deal.